app/connectors/httpParsers/ReviewHttpParser.scala

```diff
@@ -60,23 +61,33 @@ object ReviewHttpParser extends HttpParser {
           Left(MalformedResponseError)
       }
     case (_, _, response) if response.status == NOT_FOUND =>
-      checkNotFoundResponse(response)
+      Left(checkErrorResponse(response))
     case _ =>
       logger.error(s"Received service unavailable response from external-guidance. Service could be having issues.")
       Left(InternalServerError)
   }
 
-  private def checkNotFoundResponse(response: HttpResponse) = {
+  private def extractError(response: HttpResponse): RequestOutcome[Error] = {
     response.json.validate[Error] match {
-      case JsSuccess(error, _) =>
-        if (error.code == NotFoundError.code) {
-          Left(NotFoundError)
-        } else {
-          Left(StaleDataError)
-        }
+      case JsSuccess(error, _) => Right(error)
       case JsError(_) =>
-        logger.error(s"Unable to parse NOT_FOUND response from external-guidance.")
+        logger.error(s"Unable to parse response from external-guidance. Json received: ${response.json}")
```

Comment:
Might be worth mentioning this is the error response in case it is not clear from the Json.

Reply:
I'm making a few changes to the parsers for the Event changes, so will pick it up then